### PR TITLE
Add a test demonstrating how to use the django global settings.

### PR DESCRIPTION
**Pull request by mbaechtold**

### CODE REVIEW

File: tests/test_django_default_settings.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import cbs
+
+
+class MySettings(object):
+    PROJECT_NAME = 'tests'
+
+    @property
+    def INSTALLED_APPS(self):
+        return default.INSTALLED_APPS + ['test',]
```
Comment (funkybob) [inline tests/test_django_default_settings.py]: If you look at the history of this test I think you'll find it used to try a different setting... to ensure it actually extended the original.

Comment (mbaechtold) [inline tests/test_django_default_settings.py]: Yes, what about extending the `CACHES` setting in the test?

Comment (funkybob) [inline tests/test_django_default_settings.py]: Need to test an empty setting, and a setting with a default.

Comment (mbaechtold) [inline tests/test_django_default_settings.py]: Done.

File: tests/test_django_default_settings.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import unittest
+
+from django.conf import global_settings as default
+
+import cbs
+
+
+class MySettings(object):
```
Comment (funkybob) [inline tests/test_django_default_settings.py]: This is Python3 now... we don't need (object) :)

File: tests/test_django_default_settings.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    PROJECT_NAME = 'tests'
```
Comment (funkybob) [inline tests/test_django_default_settings.py]: We no longer need PROJECT_NAME.

File: tests/test_django_default_settings.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        return caches
+
+
+class GlobalSettingsTest(unittest.TestCase):
+
+    def test_precedence_empty_global_setting(self):
+        g = {}
+        cbs.apply(MySettings, g)
+
+        self.assertEqual(['test', ], g['INSTALLED_APPS'])
```
Comment (funkybob) [inline tests/test_django_default_settings.py]: trailing space

File: tests/test_django_default_settings.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+    @property
+    def INSTALLED_APPS(self):
+        # Customize an empty global setting.
+        return list(default.INSTALLED_APPS) + ['test',]
+
+    @property
+    def CACHES(self):
+        # Customize a non-empty global setting.
+        caches = default.CACHES
```
Comment (funkybob) [inline tests/test_django_default_settings.py]: I wonder if we should encourage people to .copy() the default dicts...

File: tests/test_django_default_settings.py (hunk 4 above)
Comment (mbaechtold) [inline tests/test_django_default_settings.py]: Done.

File: tests/test_django_default_settings.py (hunk 2 above)
Comment (mbaechtold) [inline tests/test_django_default_settings.py]: Done.

File: tests/test_django_default_settings.py (hunk 3 above)
Comment (mbaechtold) [inline tests/test_django_default_settings.py]: Done.

File: tests/test_django_default_settings.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import unittest
+
+from django.conf import global_settings as default
+
+import cbs
+
+
+class MySettings():
```
Comment (funkybob) [inline tests/test_django_default_settings.py]: no ()

Comment (mbaechtold) [inline tests/test_django_default_settings.py]: Done.
